fspiop/v1_0/openapi3/components/schemas/Party.yaml

```diff
@@ -0,0 +1,24 @@
+title: Party
+type: object
+description: Data model for the complex type Party.
+properties:
+  partyIdInfo:
+    $ref: ./PartyIdInfo.yaml
+    description: 'Party Id type, id, sub ID or type, and FSP Id.'
+  merchantClassificationCode:
```

Comment:
in other places, merchantClassifiactionCode is restricted to have only four digits...

Reply:
yes merchantClassifiactionCode is different from FSP ID, sorry I'm not understanding the issue with that